### DISCUSSION TITLE
OpenStreetMap in Yetiforce

### DISCUSSION THREAD

**Discussion by Aaqib-kk** [YetiForceCompany/YetiForceCRM#17299] [How to]
I have an account for OpenStreetMap, can i integrate my account in yetiForce?

**Comment by blu-IT** [comment]
As I know, for the moment the is only the "YetiForce Map" module/plugin that already works with OSM and has to be paid (5 € / month).

**Comment by ghost** [comment]
You can add your own custom tile and routing sources to the following file config/Modules/OpenStreetMap.php. Then select them under integration -> maps

**Reply by Aaqib-kk** [reply]
Yes, i checked this also. Whenever i add any custom tile and routing sources i gets Gray screen on MAP. Can you please provide an open custom tile and routing sources please?
